content/docs/02.tutorial/02.inputs.md

```diff
@@ -12,7 +12,7 @@ Inputs can be accessed in any task using a special `{{ inputs.input_name }}` [va
 
 ## Defining inputs
 
-Similarly to tasks, inputs is a list of key-value pairs. Each input must have a `name` and a `type`. You can also set `defaults` for each input. Setting default values for an input is always recommended, especially if you want to run your flow on a schedule.
+Similar to task, input is a list of key-value pairs. Each input must have a `name` and a `type`. You can also set `defaults` for each input. Setting default values for an input is always recommended, especially if you want to run your flow on a schedule.
```

Comment:
need to keep `tasks` and `inputs` as we mention the property

Reply:
Makes sense.